config/jobs/kubernetes/sig-aws/sig-aws-config.yaml

```diff
@@ -105,6 +105,34 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- interval: 1h
+  name: ci-kubernetes-e2e-kops-aws-coredns
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  spec:
+    kubeDNS:
+      provider: CoreDNS
```

Comment:
I don’t think this will pass in correctly, we need to pass this in as kops args.